src/azul/service/manifest_service.py

```diff
@@ -510,7 +510,8 @@ def _get_entities(self, path: str, doc: JSON) -> List[JSON]:
         return entities
 
     def _drs_url(self, file):
-        return self.repository_plugin.drs_uri(file['drs_path'])
+        drs_path = file['drs_path']
+        return None if drs_path is None else self.repository_plugin.drs_uri(drs_path)
```

Comment:
I think this could be a tad more readable if you do it in a if-else branch.

Reply:
I disagree that it would be more readable, but I will make the change anyway because I'm not 100% convinced that returning `None` is the correct think to do when the DRS path is not available, and the diff will look cleaner if that branch is on its own line.